sympy/physics/hydrogen.py

```diff
@@ -82,6 +83,64 @@ def R_nl(n, l, r, Z=1):
     # C =  S(2)/n**2 * sqrt(1/a**3 * factorial(n_r) / (factorial(n+l)))
     return C * r0**l * assoc_laguerre(n_r, 2*l + 1, r0).expand() * exp(-r0/2)
 
+def Psi_nlm(n, l, m, r, phi, theta, Z=1):
```

Comment:
You should retain original authorship. Create a branch based on the PR branch [der-blaue-elefant/sympy:normalised_hydrogen_wavefunctions](https://github.com/der-blaue-elefant/sympy/tree/normalised_hydrogen_wavefunctions), use `git cherry-pick`, or use `git commit --author`.

Reply:
Thank you for explaining - should be correct now.